src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -774,7 +820,14 @@ class ActivityResourceIntTest : IntegrationTestBase() {
         .expectStatus().isCreated
         .expectBody(CreateActivityResponse::class.java)
         .returnResult().responseBody
-        ?.courseActivityId
+        ?.courseActivityId!!
+
+      // Emulate the Nomis trigger COURSE_ACTIVITIES_T2.trg
+      repository.runInTransaction {
+        val activity = repository.lookupActivity(activityId)
+        activity.area = CourseActivityAreas(activityId, activity, "AREA")
+        repository.activityRepository.save(activity)
```

Comment:
You dont need the save() do you ?

Reply:
It appears that I do... must be related to transactions?